source/api.rst

```diff
@@ -0,0 +1,71 @@
+.. toctree::
+
+API
+---
+
+It's possible to do all operations using the Onedata API.
+
+The official documentation is: https://onedata.org/#/home/api.
+
+.. important:: In order to be able to access the `Onedata APIs`, it's required
+    to generate an access token.
+
+Getting an API access token
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Tokens have to be generated from the `DataHub` (Onezone) interface as
+documented in :ref:`auth-token-using-web-interface` or using a command line
+call as documented hereafter.
+
+The following variables can be exported:
+
+* `OIDC_TOKEN`: OpenID Connect Access token, see
+  https://wiki.egi.eu/wiki/Federated_Cloud_OpenStack_Providers#Obtaining_an_access_token
+  for obtaining it.
+* `ONEZONE_HOST`: name or IP of the Onezone host (to use Onezone API).
+* `ONEPROVIDER_HOST`: name or IP of the Oneprovider host (to use Oneprovider API).
+
+.. code-block:: console
+
+   export ONEZONE_HOST=datahub.egi.eu
+   export OIDC_TOKEN=<OIDC_ACCESS_TOKEN>
+   curl -H "X-Auth-Token: egi:$OIDC_TOKEN" -X POST \
+     -H 'Content-type: application/json' -d '{}' \
+     https://$ONEZONE_HOST/api/v3/onezone/user/client_tokens
+
+Registering a Handle for a file
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Prerequisites: access to a Handle service registered in the Onezone.
+See
+https://onedata.org/#/home/documentation/doc/using_onedata/handle_services.html
+for documentation on registering a new Handle service or ask a Onezone
+administrator to authorize you to use an existing Handle service already
+registered in the Onezone.
+
+
+The following variables should be exported:
+
+* `API_ACCESS_TOKEN`: Onedata API access token
```

Comment:
If at all possible, cross-reference Onedata API access token to it's definition somewhere.

Reply:
Done.